### DISCUSSION TITLE
How to customize (hide) GitLens menu & toolbar additions?

### DISCUSSION THREAD

**Discussion by heartacker** [gitkraken/vscode-gitlens#2196] [Q&A] [answered]
I am not pro
And I do not need this
![image](https://user-images.githubusercontent.com/1876302/188532739-ed66b423-9d95-4683-a08b-44f63d879da5.png)

**Comment by eamodio** [comment, marked as answer]
To customize the GitLens menus & toolbars, open the GitLens interactive settings editor from the Command Palette (<kbd>F1</kbd> or <kbd>ctrl/cmd</kbd>+<kbd>shift</kbd>+<kbd>p</kbd>) and search for _GitLens: Open Settings_. Then go down to the `Menu & Toolbars` section and toggle the desired options (e.g. the icon in your image is the option under "Add to the Source Control repository toolbar"):

![image](https://user-images.githubusercontent.com/641685/188542328-bdf7bd5c-6698-4ef7-a0ee-0594dc0fe24f.png)

For the Commit Graph specifically there is also an additional `gitlens.graph.statusBar.enabled` setting to hide the icon in the status bar.
